### ISSUE TITLE
bitbucket pipeline doesn't seem to be testing anything

### ISSUE THREAD

**Issue by joefowler** [usnistgov/mass#94]
**[Original report](https://bitbucket.org/joe_fowler/mass/issues/94) by Galen O'Neil (Bitbucket: [oneilg](https://bitbucket.org/oneilg), GitHub: [oneilg](https://github.com/oneilg)).**

----------------------------------------

Here I pasted the log from a recent test.
```
----------------------------------------------------------------------
Ran 0 tests in 0.000s
OK
('rootdir', '/opt/atlassian/pipelines/agent/build')
 ./runtests.py
found the following modules to test:
[]
```

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

No wonder we passed them all! Will you work on this one?

**Comment by joefowler** [comment]
**Original comment by Galen O'Neil (Bitbucket: [oneilg](https://bitbucket.org/oneilg), GitHub: [oneilg](https://github.com/oneilg)).**

----------------------------------------

yes, though not immediatley

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Fixed by @oneilg in 2f89c16
